jwt/validate_test.go

```diff
@@ -256,3 +257,63 @@ func TestGHIssue10(t *testing.T) {
 		}
 	})
 }
+
+type validateFn func(interface{}) error
+
+func (v validateFn) Validate(c interface{}) error {
+	return v(c)
+}
+
+func TestClaimValidator(t *testing.T) {
+	t.Parallel()
+	t.Run("interface{}", func(t *testing.T) {
+		t.Parallel()
+		err0 := errors.New("not a map")
+		err1 := errors.New("k does not exist")
+		err2 := errors.New("v is not v")
+		fn := validateFn(func(c interface{}) error {
+			m, ok := c.(map[string]interface{})
+			if !ok {
+				return err0
+			}
+			v, ok := m["k"]
+			if !ok {
+				return err1
+			}
+			if v != "v" {
+				return err2
+			}
+			return nil
+		})
+		t.Run("object is expected", func(t *testing.T) {
+			t.Parallel()
+			t1 := jwt.New()
+			assert.NoError(t, t1.Set("object", map[string]interface{}{"k": "v"}))
```

Comment:
nit: it may not happen now, but I have been trained to avoid linter warnings ... and I think using the string literal "object" over and over may cause linters to start yelling at me. Can you make this a const, like `const testClaimKey = "object"` and use that throughout